numerics/polynomial.hpp

```diff
@@ -95,6 +95,16 @@ class Polynomial {
   template<template<typename, typename, int> typename Evaluator>
   static not_null<std::unique_ptr<Polynomial>> ReadFromMessage(
       serialization::Polynomial const& message);
+
+ protected:
+  // We need to dispatch both on the subclass of Polynomial (in practice, on the
+  // degree) and on the availability of FMA, so we do both at the same time.
```

Comment:
The part regarding FMA confused me a bit: it's not dispatching unless it comes from the Dispatchagne region in France; otherwise it's just bubbly if statements.